AmplifyPlugins/Analytics/AWSPinpointAnalyticsPluginIntegrationTests/AWSPinpointAnalyticsPluginIntegrationTests.swift

```diff
@@ -149,4 +177,17 @@ class AWSPinpointAnalyticsPluginIntergrationTests: XCTestCase {
         let awsPinpoint = pinpointAnalyticsPlugin.getEscapeHatch()
         XCTAssertNotNil(awsPinpoint)
     }
+
+    private func escapeHatch() -> AWSPinpoint {
+        guard let plugin = try? Amplify.Analytics.getPlugin(for: "awsPinpointAnalyticsPlugin"),
+              let analyticsPlugin = plugin as? AWSPinpointAnalyticsPlugin else {
+            fatalError("Unable to retrieve configuration")
+        }
+        return analyticsPlugin.getEscapeHatch()
+    }
+
+    private func lowLevelClient(from configuration: AWSServiceConfiguration) -> AWSPinpointTargeting {
+        AWSPinpointTargeting.register(with: configuration, forKey: "integrationTestsTargetingConfiguration")
+        return AWSPinpointTargeting.init(forKey: "integrationTestsTargetingConfiguration")
+    }
```

Comment:
Same as above, recommend we confirm with @ruisebas or @phantumcode to confirm this should be brought into dev-preview.

Reply:
per @ruisebas, this needs to be removed.